wesnoth-units.po

```diff
@@ -6011,10 +6022,11 @@ msgstr ""
 "a staženi pod vodu, aby čelili obludě, které chapadla patří, ať už je "
 "jakákoliv."
 
+# Nedával jsem divočák, protože je v textu popsán domestikovaný chov, který víc sedí na prase bradavičnaté.
 #. [unit_type]: id=Gorer, race=monster
 #: data/core/units/monsters/Tusker_Gorer.cfg:4
 msgid "Gorer"
-msgstr ""
+msgstr "Prase bradavičnaté"
```

Comment:
tady mi mluvíte z duše - takhle by to mělo vypadat

tedy v češtině je oproti angličtině první jméno rodové a druhé druhové

u některých wesnothských potvor to druhové a rodové anglické přímo svítí a my je překládáme ve stejném pořadí, což sice ladí s ostatními jednotkami, ale ne s tím, jak jsme zvyklí pojmenovávat zvířata my Češi

tak to prosím zvažte u vašich dalších příspěvků

Reply:
No také se musí rozlišovat, jestli se v konkrétním případě jedná o skutečné druhové jméno nebo přívlastek. Např. ledový drak nemusí být drak ledový, ale drak, který je známý svým ledových charakterem.
Asi bysme potřebovali nějaký seznam návrhů a ten si projet a pobavit se, kde před a kde za.

Reply:
Ale napadá mě třeba ještěr bažinný a medvěd jeskynní. Tyto dva upravím a nad dalšími se zamyslím až při dalším updatu.

Reply:
S drakem máte pravdu, toho jsem ani nemyslel. Spíš zvířata, která mají velmi blízko těm skutečným a jejich taxonomické názvy jsou nám (Čechům) bližší než ty wesnothské. Toho [medvěda jeskynního](https://cs.wikipedia.org/wiki/Medv%C4%9Bd_jeskynn%C3%AD) jste trefil.